components/TextFields/src/private/MDCTextInputCommonFundament.m

```diff
@@ -301,6 +301,7 @@ - (void)setupUnderlineLabels {
     _leadingUnderlineLabel.textColor = MDCTextInputDefaultPlaceholderTextColor();
     _leadingUnderlineLabel.font = _textInput.font;
     _leadingUnderlineLabel.textAlignment = NSTextAlignmentNatural;
+    _leadingUnderlineLabel.lineBreakMode = NSLineBreakByWordWrapping;
```

Comment:
Is this always what clients will want by default? Should this be something that clients are expected to configure instead?